apps/client/src/pages/dashboard/apis/articleApi.ts

```
@@ -0,0 +1,83 @@
+} from '@pages/dashboard/types/api';
+
+export const getArticleDetail = async (
+  articleId: number
+): Promise<ApiResponse<ArticleDetail>> => {
+  try {
+    const { data } = await apiRequest.get<ApiResponse<ArticleDetail>>(
+      `/api/v1/articles/${articleId}`
+    );
+    return data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)